web/client/utils/cesium/BILTerrainProvider.js

```diff
@@ -963,10 +950,11 @@ module.exports = function(Cesium) {
 					resultat.getHeightmapTerrainDataImage=function(x,y,level){
 						var retour;
 						if(!isNaN(x+y+level)){
-							var urlArray=templateToURL(resultat.URLtemplateImage(x,y,level),x, y, level,provider);
+							var url=templateToURL(resultat.URLtemplateImage(x,y,level),x, y, level,provider);
 							var limitations={highest:resultat.highest,lowest:resultat.lowest,offset:resultat.offset};
+                            var proxy = resultat.proxy || { getURL: v => v } ;
 							var hasChildren = terrainChildrenMask(x, y, level,provider);
-							var promise = Cesium.throttleRequestByServer(urlArray,Cesium.loadImage);
+                            var promise = Cesium.throttleRequestByServer(proxy.getURL(urlArray),Cesium.loadImage);
```

Comment:
Is this correct? You renamed urlArray to url above, so I would expect url to be used here

Reply:
Probably I missed to save before committing. Thank you, nice catch. Tomorrow I'll try again this fix and merge this suggestion

```suggestion
                            var promise = Cesium.throttleRequestByServer(proxy.getURL(url),Cesium.loadImage);
```